mscore/menus.cpp

```diff
@@ -1187,8 +1187,8 @@ void MuseScore::showPalette(bool visible)
             updateIcons();
 
             paletteWidget = new PaletteWidget(getPaletteWorkspace(), getQmlUiEngine(), this);
-            QAction* a = getAction("toggle-palette");
-            connect(paletteWidget, &PaletteWidget::visibilityChanged, a, &QAction::setChecked);
+            QAction* a_ = getAction("toggle-palette");
```

Comment:
`a_` equals to the `a` above. Both are obtained by calling `getAction("toggle-palette");` (see line 1173).

Reply:
So dropping `QAction* ` here?

Reply:
Try it and check whether something is broken :) Looks safe, but still.